client/ratings_reviews/components/ChracteristicBreakdown.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
+import helpers from '../helpers.js';
 
-const CharacteristicBreakdown = (props) => (
-  <div>
-    <label>{props.characteristic.name}</label><br></br>
-    <meter min='0' max='5' low='1' high='4' optimum='2.5' value={props.characteristic.value}></meter>
-  </div>
-);
+const CharacteristicBreakdown = (props) => {
+  if (props.characteristic.value !== null) {
+    return (
+      <div>
+        <label>{props.characteristic.name}</label><br></br>
+        <div class='review-product-breakdown-container'>
+          <div>
+            <input class='review-product-breakdown-bar' type='range' min='1' max='5' value={props.characteristic.value}></input>
+          </div>
+          <div class='review-product-breakdown-description-container'>
+            {[1, 3, 5].map(key =>
+              <span class='review-product-breakdown-description'>{helpers.productCharacteristics[props.characteristic.name][key]}</span>
+            )}
+          </div>
+        </div>
+      </div>
+    );
+  } else {
```

Comment:
You don't need this else because you would have returned in the above if block already.

Reply:
Ah yes, thank you for pointing this out! Bad habit of mine.